test/integration/ldp.js

```diff
@@ -126,10 +140,34 @@ describe('LDP', function () {
       })
     })
 
-    it('should fail if a trailing `/` is passed', function (done) {
-      var stream = stringToStream('hello world')
-      ldp.put('localhost', '/resources/', stream, function (err) {
-        assert.equal(err.status, 409)
+    it('should create a new container', done => {
+      const containerMeta = '<> dcterms:title "Home loans".'
+      const stream = stringToStream(containerMeta)
+
+      ldp.put('localhost', '/resources/new-container/', stream, (err, status) => {
+        if (err) { return done(err) }
+
+        assert.equal(status, 201)
+
+        let written = read('new-container/.meta')
+        assert.equal(written, containerMeta)
+
+        done()
+      })
+    })
+
+    it('should update existing container meta', done => {
```

Comment:
I'd much prefer if we didn't share state between tests. Having a test rely on the output of a previous test introduces a possible race condition, especially since, depending on the test runner, tests may be run concurrently.